test/e2e/terraform/packer/consul-ami/consul.json

```diff
@@ -0,0 +1,60 @@
+{
+  "min_packer_version": "1.5.4",
+  "variables": {
+    "aws_region": "us-east-1",
+    "consul_version": "1.5.1",
+    "download_url": "{{env `CONSUL_DOWNLOAD_URL`}}",
+    "dd_api_key": "********"
```

Comment:
Might be helpful to specify this as a [sensitive variable ](https://www.packer.io/docs/templates/user-variables.html#sensitive-variables)